src/dashboard_export.py

```
@@ -0,0 +1,486 @@
+        The list of written file paths, in section order (meta, forecast,
+        metrics, anomalies, shap).
+
+    Raises:
+        jsonschema.ValidationError: If any built section fails schema validation.
+    """
+    if data_status != "sample":
+        raise ValueError(
+            "Only 'sample' data can be exported. The real-data reader is not "
+            "implemented, so recording data_status='real' would present synthetic "
+            "sample values as real model output."
+        )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)